editing_and_imputation/imputation/ratio_of_means/methodological_specification.md

```diff
@@ -59,11 +70,15 @@ multiplied by the variable of interest (auxiliary variable in the case of
 construction) for the non-respondent from a previous/consecutive/current period
 for forwards/backwards/construction imputation respectively.
 
+Due to its robust nature, it does not use any form of trimming or outliering.
+
 ## 4.0 Assumptions
 
 * The contributor's target variable value in the predictive period and the
 auxiliary variable are well correlated (i.e. a good predictor) with the
 target period.
+* The auxiliary variable is populated for every contributor when performing
```

Comment:
As mentioned in 5.0 Method Input and Output and based on my understanding. The 'auxiliary variable' should be a column in the incoming dataframe/data, and the column name should be supplied into the method using the 'auxiliary_col' option. However, reading this sentence, it appears that a 'auxiliary variable' will be generated in the impuation method while performing construction imputation. May this is one of special case which i am not aware ? or Do we need to hi-light  like 'auxiliary variable' should always  be present in inputdataframe/data